tests/xfstests/install.pm

```diff
@@ -45,6 +45,16 @@ sub run {
     zypper_call('--gpg-auto-import-keys ref', timeout => 600);
     zypper_call('in qa_test_xfstests',        timeout => 1200);
 
+    # Install upstream fio if not find usable fio in current repos
+    if (script_run('zypper se -x fio') == 0) {
+        zypper_call('in fio');
+    }
+    else {
+        send_key 'ret';
+        script_run('git clone git://git.kernel.dk/fio.git');
+        script_run('cd fio && ./configure && make && make install && cd ..');
```

Comment:
This part with git cloning and compile must be removed. There is no verification runs which could show that is something wrong with fio available in sles.

Reply:
OK, removed, but I need to keep `send_key 'ret';` to solve the contact when not found package.

Reply:
@seo @jlausuch  
Thanks for changing code to use of fio from SLES, but now you don't need a check for its availability. You can replace this part:
```
    # Install upstream fio if not find usable fio in current repos
    if (script_run('zypper se -x fio') == 0) {
        zypper_call('in fio');
    }
    else {
        send_key 'ret';
    }
```
By  fio only installation:
```
        zypper_call('in fio');
```

Because if there is no fio, it is a product bug and test should fail here. Current check would silently pass. I did research of existing bugs and missing fio was real sle15-sp1 bug https://bugzilla.suse.com/show_bug.cgi?id=1129706.

Reply:
Sorry @czerw, I'm not agree with this. In any time we need to know what we are doing, if not we will lost in plant of code and features in kernel. Here the goal is in the third line of this testscript
`# Summary: Install xfstests`
That means the goal of this test is to install xfstests, not to test if fio exist or not.(actually if fio missing, 99.5% tests will still run). To makesure xfstests got better installed here, and not like fio to block xfstests. It's the basic check when you install test package. wdyt.

Reply:
Suggested code doesn't make much sense, commentary is wrong as upstream fio is not installed anymore, it silently passes without any notice.

Reply:
Also, if xfstests needs fio it should be fixed in xfstests packaged and fio should be there as dependency https://github.com/SUSE/qa-testsuites/blob/master/tests/qa_test_xfstests/qa_test_xfstests.spec . It means that this PR is completely redundant and should be probably closed.

Reply:
this package was set requirement to fio, but it install qa_test_fio not fio, so I just removed it some days ago https://github.com/SUSE/qa-testsuites/pull/753. then I have to keep install fio by zypper.